examples/modular-drivers/Cargo.toml

```diff
@@ -1,10 +1,9 @@
 [package]
-name = "micro-rdk-esp32-sensor-examples"
+name = "modular-drivers"
```

Comment:
I'll edit this in the next PR (to separate "moving files" and editing files)

Reply:
Sorry, I was perhaps unclear. I'm not proposing moving any files here: just saying `name = micro-rdk-modular-driver-example`. So it is just an edit and should be fine to include here.

Also, FYI: Usually our workflow is to have the author of a comment resolve the comment when they feel it has been addressed, rather than resolving them yourself. Otherwise it can be hard for a reviewer to keep track of whether things changed as desired or not. I'll re-open some of the conversations that I think are still live.

Reply:
oops, thanks for the workflow info!